src/toast/_libtoast/ops_mapmaker_utils.cpp

```diff
@@ -314,12 +314,9 @@ void init_ops_mapmaker_utils(py::module & m) {
                                 use_det_flags
                             );
 
-                            // #pragma omp critical
-                            {
-                                for (int64_t iw = 0; iw < nnz; iw++) {
-                                    #pragma omp atomic
-                                    raw_zmap[zoff + iw] += zmap_val[iw];
-                                }
+                            for (int64_t iw = 0; iw < nnz; iw++) {
```

Comment:
This is also changed in #652, so I'll deal with that conflict after this is merged.